frame/evm/src/lib.rs

```diff
@@ -322,6 +328,36 @@ pub mod pallet {
 				pays_fee: Pays::No,
 			})
 		}
+
+		/// Increment `sufficients` for existing accounts having a nonzero `nonce` but zero `sufficients` value.
+		#[pallet::weight(
+			<T as pallet::Config>::WeightInfo::hotfix_inc_account_sufficients(addresses.len().try_into().unwrap_or(u32::MAX))
+		)]
+		pub fn hotfix_inc_account_sufficients(
+			origin: OriginFor<T>,
+			addresses: Vec<H160>,
+		) -> DispatchResultWithPostInfo {
+			const MAX_ADDRESS_COUNT: usize = 1000;
+
+			frame_system::ensure_signed(origin)?;
+			ensure!(
+				addresses.len() <= MAX_ADDRESS_COUNT,
+				Error::<T>::MaxAddressCountExceeded
+			);
+
+			for address in addresses {
+				let account_id = T::AddressMapping::into_account_id(address);
+				let nonce = frame_system::Pallet::<T>::account_nonce(&account_id);
+				if !nonce.is_zero() {
+					frame_system::Pallet::<T>::inc_sufficients(&account_id);
+				}
+			}
+
+			Ok(PostDispatchInfo {
+				actual_weight: None,
+				pays_fee: Pays::No,
```

Comment:
This is **dangerous**. anyone can execute this extrinsic and fill a block without paying any fee.
Ideally, it should be `Pays::No` only if 
```
addresses.len() > 0 && fixedAddresses.len() == addresses.len()
```